google-http-client/src/main/java/com/google/api/client/json/webtoken/JsonWebSignature.java

```diff
@@ -482,6 +482,11 @@ private static X509TrustManager getDefaultX509TrustManager() {
     return signedContentBytes;
   }
 
+  /** Returns the URL safe Base64 encoded JWS token. */
+  public String getRawToken() {
```

Comment:
Since we're unlikely to get the API perfect on the first try, we should mark this @Beta to warn users it might change.